src/components/Modeler.vue

```diff
@@ -195,7 +195,7 @@ export default {
       this.cursor = null;
     },
     scale(scale) {
-      this.paper.scale(scale);
+      this.paperManager.paper.scale(scale);
```

Comment:
Since you have a getter for `scale`, I think adding a setter for it would be nice as well. Then you can have:
```js
this.paperManager.scale = scale;
```